### ISSUE TITLE
WCVP - Hedychium gardnerianum Sheppard ex Ker Gawl. should be Hedychium gardnerianum Sheph. Ex Ker Gawl.

### ISSUE THREAD

**Issue by jhnwllr** [CatalogueOfLife/data#524]
**Describe the problem**:

A GBIF user has given the following feedback:
https://github.com/gbif/backbone-feedback/issues/147

**Hedychium gardnerianum Sheppard ex Ker Gawl.**

should be 

**Hedychium gardnerianum Sheph. Ex Ker Gawl.**

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/dataset/2232
https://www.catalogueoflife.org/data/taxon/3JZPN

**Literature references**:
Pereira, M.J., Eleutério, T., Meirelles, M.G. et al. Hedychium gardnerianum Sheph. ex Ker Gawl. from its discovery to its invasive status: a review. Bot Stud 62, 11 (2021). https://doi.org/10.1186/s40529-021-00318-5

**Comment by yroskov** [comment]
Indeed, it's a bug in the source database. The authorstring will be corrected in CoL.
